androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/currentpatients/CurrentPatientsList.kt

```diff
@@ -0,0 +1,61 @@
+package com.EENX15_22_17.digital_journal.android.ui.currentpatients
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+
+class CurrentPatientsList {
+
+    // Temporary data before backend is implemented
+    private val defaultPatients: List<CurrentPatientsData> = listOf(
+        CurrentPatientsData(
```

Comment:
This data is present in two places, make the PreviewParameterProvider use this list resource (or the other way around)